packages/core/scripts/optimistic-oracle-umip/3_Verify.js

```diff
@@ -0,0 +1,60 @@
+// This script verify that the upgrade was executed correctly.
+// It can be run on mainnet after the upgrade is completed or on the local Ganache mainnet fork to validate the
+// execution of the previous two scripts. This script does not need any wallets unlocked and does not make any on-chain
+// state changes. It can be run as:
+// truffle exec ./scripts/identifier-umip/3_Verify.js --network mainnet-fork --identifier USDETH --identifier ETHBTC
+
+const assert = require("assert").strict;
+
+// Use the same ABI's as deployed contracts:
+const { getTruffleContract } = require("../../index");
+const Governor = getTruffleContract("Governor", web3, "1.1.0");
+const Finder = getTruffleContract("Finder", web3, "1.1.0");
+const Registry = getTruffleContract("Registry", web3, "1.1.0");
+
+const { RegistryRolesEnum, interfaceName } = require("@uma/common");
+
+const argv = require("minimist")(process.argv.slice(), { string: ["deployedAddress"] });
+
+async function runExport() {
+  console.log("Running Upgrade Verifier🔥");
+
+  if (!argv.deployedAddress) {
+    throw new Error("Must specify --deployedAddress");
+  }
+
+  const registry = await Registry.deployed();
+  const governor = await Governor.deployed();
+  const finder = await Finder.deployed();
+
+  console.log("Verifying that Governor doesn't hold the creator role...");
+  assert(!(await registry.holdsRole(RegistryRolesEnum.CONTRACT_CREATOR, governor.address)));
+  console.log("Verified!");
+
+  console.log("Verifying that the OptimisticOracle is registered with the Registry...");
+  assert(await registry.isContractRegistered(argv.deployedAddress));
```

Comment:
this should just be assert. it should be `assert.true(...)` or `assert.equal(...)`.

Reply:
Does `assert.true` exist for the nodejs assert library? I only used `assert` because I wasn't familiar with any boolean checker in native nodejs.

Reply:
yea theres no `assert.true`, but `assert` and `assert.ok` are equivalent.

Reply:
sorry, I meant `assert.isTrue`

Reply:
@chrismaree Ah, I think there's minor miscommunication. The issue is that the docs you reference are from the chai assert library that is _generally_ meant to be an extension of asserts for test environments (it is automatically injected into truffle tests along with mocha). We _could_ import chai in non-test scripts, but we haven't done that in the past as far as I know. The native nodejs assert doesn't have an `isTrue` (see docs here: https://nodejs.org/docs/latest-v12.x/api/assert.html), so we would either need to use `assert()` or `assert.ok()`.

Reply:
aha. I see. `assert.isTrue` is explicitly used within unit testing, and not the general node Assert context. Fair enough! In that case, assert is the way to go :)